t4_devkit/dataclass/pointcloud.py

```diff
@@ -91,12 +229,17 @@ def num_dims() -> int:
         return 4
 
     @classmethod
-    def from_file(cls, filepath: str) -> Self:
+    def from_file(cls, filepath: str, metainfo_filepath: str | None = None) -> Self:
         assert filepath.endswith(".bin"), f"Unexpected filetype: {filepath}"
 
         scan = np.fromfile(filepath, dtype=np.float32)
         points = scan.reshape((-1, 5))[:, : cls.num_dims()]
-        return cls(points.T)
+
+        metainfo = None
+        if metainfo_filepath is not None:
+            metainfo = PointCloudMetainfo.from_file(metainfo_filepath)
```

Comment:
[NITS] For me, using ternary operator is readable and safe.
```suggestion
        metainfo = PointCloudMetainfo.from_file(metainfo_filepath) if metainfo_filepath is not None else None
```